autoscale/src/main/java/com/sequenceiq/periscope/service/RegularScalingAdjustmentService.java

```diff
@@ -73,16 +86,17 @@ private void publishScalingEventIfNeeded(Cluster cluster, int existingClusterNod
 
         if (scalingAdjustmentParameters.getUpscaleAdjustment() != null) {
             Integer targetScaleUpCount = Math.min(scalingAdjustmentParameters.getUpscaleAdjustment(), loadAlertConfiguration.getMaxScaleUpStepSize());
-
+            scalingTrigger = scalingTriggerService.create(cluster, TriggerStatus.MANDATORY_UPSCALE, null,
```

Comment:
Can the timestamp be passed in as a parameter here, and in similar instances elsewhere.

Could consider changing EvaluatorExecutor (base for all Evaluators?) to include a startTime.